framework/components/ecl-links/_links.scss

```diff
@@ -26,7 +26,10 @@
   color: #fff;
 
   &:hover,
-  &:focus,
+  &:focus {
```

Comment:
`visited` can go back here

Reply:
I kept it separated as I feel that it may change again in coming days/week. But ok, I'll put it back